CHANGELOG.md

```diff
@@ -16,19 +16,21 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Switched off broadcasting of transactions which the node considers
   incorrect. (#1781)
 
+- Fixed incorrect invalidation of block proposals. (#1782)
+
 ### Breaking changes
 
+#### exonum
+
+- `Runtime::deploy_artifact` longer returns `Box<dyn Future<...>>`. Instead a special
```

Comment:
Also, this should probably say "`Runtime::deploy_artifact` **no** longer returns..."